language/German/strings.po

```diff
@@ -691,28 +696,26 @@ msgid "Wait Vertical Sync"
 msgstr "Warten auf vertikalen Synchronisationsimpuls"
 
 #: src/frontend/mame/ui/menu.cpp:364 src/frontend/mame/ui/menu.cpp:722
-#: src/frontend/mame/ui/videoopt.cpp:169
-#: src/frontend/mame/ui/inputtoggle.cpp:96 src/frontend/mame/ui/custui.cpp:480
-#: src/frontend/mame/ui/analogipt.cpp:478 plugins/cheatfind/init.lua:766
-#: plugins/cheatfind/init.lua:777 plugins/cheat/init.lua:679
+#: src/frontend/mame/ui/videoopt.cpp:169 src/frontend/mame/ui/inputtoggle.cpp:96
+#: src/frontend/mame/ui/custui.cpp:480 src/frontend/mame/ui/analogipt.cpp:478
+#: plugins/cheatfind/init.lua:766 plugins/cheatfind/init.lua:777
+#: plugins/cheat/init.lua:679
 msgid "On"
 msgstr "An"
 
 #: src/frontend/mame/ui/menu.cpp:364 src/frontend/mame/ui/menu.cpp:725
-#: src/frontend/mame/ui/videoopt.cpp:185
-#: src/frontend/mame/ui/inputtoggle.cpp:91 src/frontend/mame/ui/custui.cpp:480
-#: src/frontend/mame/ui/analogipt.cpp:478 plugins/cheatfind/init.lua:763
-#: plugins/cheatfind/init.lua:774 plugins/cheat/init.lua:682
-#: plugins/cheat/init.lua:691
+#: src/frontend/mame/ui/videoopt.cpp:185 src/frontend/mame/ui/inputtoggle.cpp:91
+#: src/frontend/mame/ui/custui.cpp:480 src/frontend/mame/ui/analogipt.cpp:478
+#: plugins/cheatfind/init.lua:763 plugins/cheatfind/init.lua:774
+#: plugins/cheat/init.lua:682 plugins/cheat/init.lua:691
 msgid "Off"
 msgstr "Aus"
 
 #: src/frontend/mame/ui/menu.cpp:728
 msgid "Auto"
 msgstr "Auto"
 
-#: src/frontend/mame/ui/menu.cpp:1233
-#: src/frontend/mame/ui/simpleselgame.cpp:283
+#: src/frontend/mame/ui/menu.cpp:1233 src/frontend/mame/ui/simpleselgame.cpp:283
```

Comment:
Can you please back out these changes re-wrapping the source locations?  It’s extra churn and makes the files not line up across languages.

Reply:
I'm not aware of doing any of these changes. 
I'm just using Poedit to do the translations.
I'll try to undo them though.